stock_prices/stock_prices.py

```diff
@@ -2,14 +2,33 @@
 
 import argparse
 
+
 def find_max_profit(prices):
-  pass
+    max_profit = 0
+    i = 0
+
+    while i < len(prices)-1:
+        current_price = prices[i]
+        current_max = max(prices[i+1:])
+
```

Comment:
hah, did you get this from the Q and A or from your own musings? either way, awesome work. I never was able to get this on my first try; I'm impressed at how clean your code is! A+